Utilities/CodeGeneration/EdFi.Ods.CodeGen.Tests/Approval/5.2.0/DataStandard_520_ApprovalTests.Verify.Extensions.Sample.1.0.0_Std_5.2.0_Models_Entities_Entities.generated.approved.cs

```diff
@@ -629,6 +660,7 @@ public virtual int? StaffClassificationDescriptorId
         private int? _staffClassificationDescriptorId;
         private string _staffClassificationDescriptor;
 
+        [IgnoreMember]
```

Comment:
We serialize descriptor Ids, not the full URI. It's definitive and more compact.